.github/workflows/bot-coderabbit-plan-trigger.yml

```
@@ -0,0 +1,49 @@
+name: CodeRabbit Plan Trigger
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)